### PR TITLE
Update LTI 1.3 integration

### PR DESCRIPTION
**Pull request by vutoan1245**

### CODE REVIEW

File: backend/src/handlers/auth/lti.ts [hunk 1]
```diff
@@ -9,3 +14,51 @@ export const launch: APIGatewayProxyHandler = async () => {
...
+	console.log('--------------------\nlogin');
+
+	// TODO: generate v4 uuid;
+	const state = 'f0e41f9d-dc90-471b-a651-636ae26df124';
+	const nonce = '75fd1447-cd57-4423-bc3b-a11040b9adb9';
+
+	const body = querystring.parse(event.body as string);
+	console.log('body', body);
+
+	const url =
```
Comment (kennygperez) [inline backend/src/handlers/auth/lti.ts:30]: that's a long url lol

File: backend/serverless.yml [hunk 2]
```diff
@@ -215,6 +215,32 @@ functions:
           path: /api/v1/auth/lti
           method: post
           cors: true
+  # Step 2: Authentication Request
+  login:
+    handler: src/handlers/auth/lti.login
+    timeout: 10
+    events:
+      - http:
+          path: /api/v1/auth/login
```
Comment (kennygperez) [inline backend/serverless.yml:224]: Fix the paths to follow the following convention

`api/v1/auth/{PLATFORM}/{ACTION}`

**Review by kennygperez — CHANGES_REQUESTED** [review]
Change URL schema

**Review by ljoks — DISMISSED** [review]

**Review by kennygperez — APPROVED** [review]
Niceeee

**Comment by kennygperez** [comment]
@vutoan1245 merge when ready

**Pull request merged**